.github/workflows/ci.yaml

```diff
@@ -21,9 +39,24 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install --upgrade ansible
 
-      - name: Run the all-in-one playbook
+      - name: Create block storage device (volume)
+        run: |
+          VOLUME_DEVICE="$(losetup -f)"
+          VOLUME_IMAGE="${{ runner.temp }}/diskimage"
+
+          dd if=/dev/zero of=$VOLUME_IMAGE bs=1M count=1024
+          sudo losetup $VOLUME_DEVICE $VOLUME_IMAGE
```

Comment:
@malor I figured that we can create a loop back device and test our `volume` role too. :)

Reply:
Good point! Not sure why I did not think of this before! :(